BookStoreProd/src/main/java/mate/academy/bookstoreprod/service/impl/BookServiceImpl.java

```diff
@@ -35,4 +36,15 @@ public BookDto findById(Long id) {
         return bookMapper.toBookDto(bookRepository.findById(id)
                 .orElseThrow(() -> new EntityNotFoundException("Book not found with id: " + id)));
     }
+
+    @Override
+    public void updateById(Long id, UpdateBookRequestDto dto) {
+        Book book = bookMapper.toBook(dto);
+        bookRepository.updateById(id, book);
```

Comment:
```suggestion
        bookMapper.updateBookFromDto(bookDto, book);
```
save book
return BookDto